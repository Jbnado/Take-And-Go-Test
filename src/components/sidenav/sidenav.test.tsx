import TestRenderer from "react-test-renderer";
import { describe, it, expect } from "vitest";
import { Sidenav } from "./index";
import { ChevronLeft } from "react-bootstrap-icons";

const renderComponent = (props = {}) =>
  TestRenderer.create(<Sidenav {...props} />);

describe("Sidenav Component", () => {
  it("renders correctly when open", () => {
    const tree = renderComponent().toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders correctly when closed", () => {
    const tree = renderComponent({ isOpen: false }).toJSON();
    expect(tree).toMatchSnapshot();
  });

  describe("changes state on click", async () => {
    const component = renderComponent();
    const waitAanimation = await new Promise((resolve) =>
      setTimeout(resolve, 500)
    );

    it("should start opened", () => {
      const aside = component.root.findByType("aside");
      expect(aside.props.className).toContain("w-64");
    });

    it("should close on click", async () => {
      const chevron = component.root.findByType(ChevronLeft);
      chevron.props.onClick();
      await waitAanimation;
      const aside = component.root.findByType("aside");
      expect(aside.props.className).toContain("w-20");
    });

    it("should open on click", async () => {
      const chevron = component.root.findByType(ChevronLeft);
      chevron.props.onClick();
      await waitAanimation;
      const aside = component.root.findByType("aside");
      expect(aside.props.className).toContain("w-64");
    });
  });
});
