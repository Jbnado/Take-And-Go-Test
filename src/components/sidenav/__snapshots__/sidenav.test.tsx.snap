// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sidenav Component > renders correctly when closed 1`] = `
<aside
  className=" w-20 p-6 h-screen relative duration-300 shadow-md"
>
  <svg
    className="absolute cursor-pointer -right-3 top-9 w-7
           border rounded-full  rotate-180"
    fill="currentColor"
    height="1em"
    onClick={[Function]}
    viewBox="0 0 16 16"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
      fillRule="evenodd"
    />
  </svg>
  <nav>
    <ul>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6 1v3H1V1h5zM1 0a1 1 0 0 0-1 1v3a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1H1zm14 12v3h-5v-3h5zm-5-1a1 1 0 0 0-1 1v3a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1v-3a1 1 0 0 0-1-1h-5zM6 8v7H1V8h5zM1 7a1 1 0 0 0-1 1v7a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V8a1 1 0 0 0-1-1H1zm14-6v7h-5V1h5zm-5-1a1 1 0 0 0-1 1v7a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1h-5z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Dashboard
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M3 2v4.586l7 7L14.586 9l-7-7H3zM2 2a1 1 0 0 1 1-1h4.586a1 1 0 0 1 .707.293l7 7a1 1 0 0 1 0 1.414l-4.586 4.586a1 1 0 0 1-1.414 0l-7-7A1 1 0 0 1 2 6.586V2z"
          />
          <path
            d="M5.5 5a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1zm0 1a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zM1 7.086a1 1 0 0 0 .293.707L8.75 15.25l-.043.043a1 1 0 0 1-1.414 0l-7-7A1 1 0 0 1 0 7.586V3a1 1 0 0 1 1-1v5.086z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Oportunidades
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M3.5 0a.5.5 0 0 1 .5.5V1h8V.5a.5.5 0 0 1 1 0V1h1a2 2 0 0 1 2 2v11a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V3a2 2 0 0 1 2-2h1V.5a.5.5 0 0 1 .5-.5zM1 4v10a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V4H1z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Agenda
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 9a1 1 0 1 1-2 0 1 1 0 0 1 2 0Zm10 0a1 1 0 1 1-2 0 1 1 0 0 1 2 0ZM6 8a1 1 0 0 0 0 2h4a1 1 0 1 0 0-2H6ZM4.862 4.276 3.906 6.19a.51.51 0 0 0 .497.731c.91-.073 2.35-.17 3.597-.17 1.247 0 2.688.097 3.597.17a.51.51 0 0 0 .497-.731l-.956-1.913A.5.5 0 0 0 10.691 4H5.309a.5.5 0 0 0-.447.276Z"
          />
          <path
            d="M2.52 3.515A2.5 2.5 0 0 1 4.82 2h6.362c1 0 1.904.596 2.298 1.515l.792 1.848c.075.175.21.319.38.404.5.25.855.715.965 1.262l.335 1.679c.033.161.049.325.049.49v.413c0 .814-.39 1.543-1 1.997V13.5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1-.5-.5v-1.338c-1.292.048-2.745.088-4 .088s-2.708-.04-4-.088V13.5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1-.5-.5v-1.892c-.61-.454-1-1.183-1-1.997v-.413a2.5 2.5 0 0 1 .049-.49l.335-1.68c.11-.546.465-1.012.964-1.261a.807.807 0 0 0 .381-.404l.792-1.848ZM4.82 3a1.5 1.5 0 0 0-1.379.91l-.792 1.847a1.8 1.8 0 0 1-.853.904.807.807 0 0 0-.43.564L1.03 8.904a1.5 1.5 0 0 0-.03.294v.413c0 .796.62 1.448 1.408 1.484 1.555.07 3.786.155 5.592.155 1.806 0 4.037-.084 5.592-.155A1.479 1.479 0 0 0 15 9.611v-.413c0-.099-.01-.197-.03-.294l-.335-1.68a.807.807 0 0 0-.43-.563 1.807 1.807 0 0 1-.853-.904l-.792-1.848A1.5 1.5 0 0 0 11.18 3H4.82Z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Veículos 
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.752 6.193c.599.6 1.73.437 2.528-.362.798-.799.96-1.932.362-2.531-.599-.6-1.73-.438-2.528.361-.798.8-.96 1.933-.362 2.532Z"
          />
          <path
            d="M15.811 3.312c-.363 1.534-1.334 3.626-3.64 6.218l-.24 2.408a2.56 2.56 0 0 1-.732 1.526L8.817 15.85a.51.51 0 0 1-.867-.434l.27-1.899c.04-.28-.013-.593-.131-.956a9.42 9.42 0 0 0-.249-.657l-.082-.202c-.815-.197-1.578-.662-2.191-1.277-.614-.615-1.079-1.379-1.275-2.195l-.203-.083a9.556 9.556 0 0 0-.655-.248c-.363-.119-.675-.172-.955-.132l-1.896.27A.51.51 0 0 1 .15 7.17l2.382-2.386c.41-.41.947-.67 1.524-.734h.006l2.4-.238C9.005 1.55 11.087.582 12.623.208c.89-.217 1.59-.232 2.08-.188.244.023.435.06.57.093.067.017.12.033.16.045.184.06.279.13.351.295l.029.073a3.475 3.475 0 0 1 .157.721c.055.485.051 1.178-.159 2.065Zm-4.828 7.475.04-.04-.107 1.081a1.536 1.536 0 0 1-.44.913l-1.298 1.3.054-.38c.072-.506-.034-.993-.172-1.418a8.548 8.548 0 0 0-.164-.45c.738-.065 1.462-.38 2.087-1.006ZM5.205 5c-.625.626-.94 1.351-1.004 2.09a8.497 8.497 0 0 0-.45-.164c-.424-.138-.91-.244-1.416-.172l-.38.054 1.3-1.3c.245-.246.566-.401.91-.44l1.08-.107-.04.039Zm9.406-3.961c-.38-.034-.967-.027-1.746.163-1.558.38-3.917 1.496-6.937 4.521-.62.62-.799 1.34-.687 2.051.107.676.483 1.362 1.048 1.928.564.565 1.25.941 1.924 1.049.71.112 1.429-.067 2.048-.688 3.079-3.083 4.192-5.444 4.556-6.987.183-.771.18-1.345.138-1.713a2.835 2.835 0 0 0-.045-.283 3.078 3.078 0 0 0-.3-.041Z"
          />
          <path
            d="M7.009 12.139a7.632 7.632 0 0 1-1.804-1.352A7.568 7.568 0 0 1 3.794 8.86c-1.102.992-1.965 5.054-1.839 5.18.125.126 3.936-.896 5.054-1.902Z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Publicação
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z"
          />
          <path
            d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"
            fillRule="evenodd"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Contatos
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h1v15h15v1H0V0Zm10 3.5a.5.5 0 0 1 .5-.5h4a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-1 0V4.9l-3.613 4.417a.5.5 0 0 1-.74.037L7.06 6.767l-3.656 5.027a.5.5 0 0 1-.808-.588l4-5.5a.5.5 0 0 1 .758-.06l2.609 2.61L13.445 4H10.5a.5.5 0 0 1-.5-.5Z"
            fillRule="evenodd"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Analytics 
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m8 0 6.61 3h.89a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-.5.5H15v7a.5.5 0 0 1 .485.38l.5 2a.498.498 0 0 1-.485.62H.5a.498.498 0 0 1-.485-.62l.5-2A.501.501 0 0 1 1 13V6H.5a.5.5 0 0 1-.5-.5v-2A.5.5 0 0 1 .5 3h.89L8 0ZM3.777 3h8.447L8 1 3.777 3ZM2 6v7h1V6H2Zm2 0v7h2.5V6H4Zm3.5 0v7h1V6h-1Zm2 0v7H12V6H9.5ZM13 6v7h1V6h-1Zm2-1V4H1v1h14Zm-.39 9H1.39l-.25 1h13.72l-.25-1Z"
          />
        </svg>
        <span
          className="hidden origin-left duration-200"
        >
          Financiamento
        </span>
      </li>
    </ul>
  </nav>
</aside>
`;

exports[`Sidenav Component > renders correctly when open 1`] = `
<aside
  className=" w-64 p-8 h-screen relative duration-300 shadow-md"
>
  <svg
    className="absolute cursor-pointer -right-3 top-9 w-7
           border rounded-full  false"
    fill="currentColor"
    height="1em"
    onClick={[Function]}
    viewBox="0 0 16 16"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
      fillRule="evenodd"
    />
  </svg>
  <nav>
    <ul>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6 1v3H1V1h5zM1 0a1 1 0 0 0-1 1v3a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1H1zm14 12v3h-5v-3h5zm-5-1a1 1 0 0 0-1 1v3a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1v-3a1 1 0 0 0-1-1h-5zM6 8v7H1V8h5zM1 7a1 1 0 0 0-1 1v7a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V8a1 1 0 0 0-1-1H1zm14-6v7h-5V1h5zm-5-1a1 1 0 0 0-1 1v7a1 1 0 0 0 1 1h5a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1h-5z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Dashboard
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M3 2v4.586l7 7L14.586 9l-7-7H3zM2 2a1 1 0 0 1 1-1h4.586a1 1 0 0 1 .707.293l7 7a1 1 0 0 1 0 1.414l-4.586 4.586a1 1 0 0 1-1.414 0l-7-7A1 1 0 0 1 2 6.586V2z"
          />
          <path
            d="M5.5 5a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1zm0 1a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zM1 7.086a1 1 0 0 0 .293.707L8.75 15.25l-.043.043a1 1 0 0 1-1.414 0l-7-7A1 1 0 0 1 0 7.586V3a1 1 0 0 1 1-1v5.086z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Oportunidades
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M3.5 0a.5.5 0 0 1 .5.5V1h8V.5a.5.5 0 0 1 1 0V1h1a2 2 0 0 1 2 2v11a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V3a2 2 0 0 1 2-2h1V.5a.5.5 0 0 1 .5-.5zM1 4v10a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V4H1z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Agenda
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 9a1 1 0 1 1-2 0 1 1 0 0 1 2 0Zm10 0a1 1 0 1 1-2 0 1 1 0 0 1 2 0ZM6 8a1 1 0 0 0 0 2h4a1 1 0 1 0 0-2H6ZM4.862 4.276 3.906 6.19a.51.51 0 0 0 .497.731c.91-.073 2.35-.17 3.597-.17 1.247 0 2.688.097 3.597.17a.51.51 0 0 0 .497-.731l-.956-1.913A.5.5 0 0 0 10.691 4H5.309a.5.5 0 0 0-.447.276Z"
          />
          <path
            d="M2.52 3.515A2.5 2.5 0 0 1 4.82 2h6.362c1 0 1.904.596 2.298 1.515l.792 1.848c.075.175.21.319.38.404.5.25.855.715.965 1.262l.335 1.679c.033.161.049.325.049.49v.413c0 .814-.39 1.543-1 1.997V13.5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1-.5-.5v-1.338c-1.292.048-2.745.088-4 .088s-2.708-.04-4-.088V13.5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1-.5-.5v-1.892c-.61-.454-1-1.183-1-1.997v-.413a2.5 2.5 0 0 1 .049-.49l.335-1.68c.11-.546.465-1.012.964-1.261a.807.807 0 0 0 .381-.404l.792-1.848ZM4.82 3a1.5 1.5 0 0 0-1.379.91l-.792 1.847a1.8 1.8 0 0 1-.853.904.807.807 0 0 0-.43.564L1.03 8.904a1.5 1.5 0 0 0-.03.294v.413c0 .796.62 1.448 1.408 1.484 1.555.07 3.786.155 5.592.155 1.806 0 4.037-.084 5.592-.155A1.479 1.479 0 0 0 15 9.611v-.413c0-.099-.01-.197-.03-.294l-.335-1.68a.807.807 0 0 0-.43-.563 1.807 1.807 0 0 1-.853-.904l-.792-1.848A1.5 1.5 0 0 0 11.18 3H4.82Z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Veículos 
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.752 6.193c.599.6 1.73.437 2.528-.362.798-.799.96-1.932.362-2.531-.599-.6-1.73-.438-2.528.361-.798.8-.96 1.933-.362 2.532Z"
          />
          <path
            d="M15.811 3.312c-.363 1.534-1.334 3.626-3.64 6.218l-.24 2.408a2.56 2.56 0 0 1-.732 1.526L8.817 15.85a.51.51 0 0 1-.867-.434l.27-1.899c.04-.28-.013-.593-.131-.956a9.42 9.42 0 0 0-.249-.657l-.082-.202c-.815-.197-1.578-.662-2.191-1.277-.614-.615-1.079-1.379-1.275-2.195l-.203-.083a9.556 9.556 0 0 0-.655-.248c-.363-.119-.675-.172-.955-.132l-1.896.27A.51.51 0 0 1 .15 7.17l2.382-2.386c.41-.41.947-.67 1.524-.734h.006l2.4-.238C9.005 1.55 11.087.582 12.623.208c.89-.217 1.59-.232 2.08-.188.244.023.435.06.57.093.067.017.12.033.16.045.184.06.279.13.351.295l.029.073a3.475 3.475 0 0 1 .157.721c.055.485.051 1.178-.159 2.065Zm-4.828 7.475.04-.04-.107 1.081a1.536 1.536 0 0 1-.44.913l-1.298 1.3.054-.38c.072-.506-.034-.993-.172-1.418a8.548 8.548 0 0 0-.164-.45c.738-.065 1.462-.38 2.087-1.006ZM5.205 5c-.625.626-.94 1.351-1.004 2.09a8.497 8.497 0 0 0-.45-.164c-.424-.138-.91-.244-1.416-.172l-.38.054 1.3-1.3c.245-.246.566-.401.91-.44l1.08-.107-.04.039Zm9.406-3.961c-.38-.034-.967-.027-1.746.163-1.558.38-3.917 1.496-6.937 4.521-.62.62-.799 1.34-.687 2.051.107.676.483 1.362 1.048 1.928.564.565 1.25.941 1.924 1.049.71.112 1.429-.067 2.048-.688 3.079-3.083 4.192-5.444 4.556-6.987.183-.771.18-1.345.138-1.713a2.835 2.835 0 0 0-.045-.283 3.078 3.078 0 0 0-.3-.041Z"
          />
          <path
            d="M7.009 12.139a7.632 7.632 0 0 1-1.804-1.352A7.568 7.568 0 0 1 3.794 8.86c-1.102.992-1.965 5.054-1.839 5.18.125.126 3.936-.896 5.054-1.902Z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Publicação
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z"
          />
          <path
            d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"
            fillRule="evenodd"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Contatos
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h1v15h15v1H0V0Zm10 3.5a.5.5 0 0 1 .5-.5h4a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-1 0V4.9l-3.613 4.417a.5.5 0 0 1-.74.037L7.06 6.767l-3.656 5.027a.5.5 0 0 1-.808-.588l4-5.5a.5.5 0 0 1 .758-.06l2.609 2.61L13.445 4H10.5a.5.5 0 0 1-.5-.5Z"
            fillRule="evenodd"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Analytics 
        </span>
      </li>
      <li
        className="flex rounded-full py-1 cursor-pointer hover:bg-light-white text-sm items-center gap-x-4 mb-5"
      >
        <svg
          fill="currentColor"
          height={20}
          viewBox="0 0 16 16"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m8 0 6.61 3h.89a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-.5.5H15v7a.5.5 0 0 1 .485.38l.5 2a.498.498 0 0 1-.485.62H.5a.498.498 0 0 1-.485-.62l.5-2A.501.501 0 0 1 1 13V6H.5a.5.5 0 0 1-.5-.5v-2A.5.5 0 0 1 .5 3h.89L8 0ZM3.777 3h8.447L8 1 3.777 3ZM2 6v7h1V6H2Zm2 0v7h2.5V6H4Zm3.5 0v7h1V6h-1Zm2 0v7H12V6H9.5ZM13 6v7h1V6h-1Zm2-1V4H1v1h14Zm-.39 9H1.39l-.25 1h13.72l-.25-1Z"
          />
        </svg>
        <span
          className="false origin-left duration-200"
        >
          Financiamento
        </span>
      </li>
    </ul>
  </nav>
</aside>
`;
